loops-marc-jacob.js

```diff
@@ -0,0 +1,104 @@
+//Challenge #1 Logging values with for loops
+// Write a for loop that logs each number from 1 - 20.
+for (let i = 1; i < 21; i++) {
+    console.log(i)
+}
+
+// Write a for loop that logs the result of each number from 1 - 20 tripled.
+for (let i = 1; i < 21; i++) {
+    console.log(i * 3)
+}
+
+// Create a for loop that logs each even number from 1-20, and in the place of every odd number,
+// returns the word "ODD" Expected output --> ODD, 2, ODD, 4, ODD, 6 ...etc
+for (let i = 1; i < 21; i++) {
+    if (i % 2 === 1 ) {
+        console.log ("odd")
+    } else {
+        console.log(i)
+    }
+}
+//----------------------------------------------------------------------------------------->
+
+
+//Challenge #2 Looping over an array. Consider this variable:
+    var nums = [3, 57, -9, 20, 67]
+
+// Create a loop that will log the highest number from the array. Expected output --> 67
+let highestNumber;
```

Comment:
The beauty of coding is that there are so many ways to obtain the desired output. Nevertheless, there are also best practices.

- Line 28 is an undefined variable if you were to console.log() the highestNumber it would be undefined at this line. I recommend assigning a value initially, such as
`let highestNumber = nums[0]`
By declaring a variable and assigning a value, you can now remove lines 30 - 32.